tests/integration/testlib/examplelib.py

```diff
@@ -41,7 +41,11 @@ def example_state(initial, cleanup=None):
         yield desired_state
     finally:
         if cleanup:
-            libnmstate.apply(load_example(cleanup))
+            try:
+                libnmstate.apply(load_example(cleanup), verify_change=True)
+            except libnmstate.error.NmstateVerificationError:
```

Comment:
Why only for the verification error? Shouldn't it try to cleanup in case of any error?

Reply:
The `verify_change=False` is only useful if all is good except of the verification step (like partial cleanup vs all-or-nothing cleanup).
If there is a code error or anything unexpected, it should blow up as we have nothing better to do.

Reply:
There could still be a coding error in the verification step. Nevertheless, the more I think about this, the cleanup as part as the tear down belongs IMHO in a fixture or decorator instead of in a context inside the test. This change makes it hard to understand its intentions.

Reply:
A fixture usually uses the contextmanager. When and where you call it is a topic by itself.
Please re-suggest a solution that will allow us to move forward with cleaning up the example tests.

Reply:
I can live with your current solution if you update the commit message. I provided a suggestion since it did not seem to be accurate or correct English.